recipe/meta.yaml

```diff
@@ -41,10 +41,11 @@ requirements:
     - proj
     - pyproj >=3.0
     - geos
-    # including matplotlib-base even though it's optional upstream
     - matplotlib-base >=3.1
     - scipy >=0.10
     - ucrt  # [win]
+    # make openssl an explicit dependency because we're having trouble with >=3
+    - openssl
```

Comment:
@conda-forge/cartopy, are we okay with adding openssl explicitly as a dependency so we can constrain in to 1.1.1 for now and drop it if 3.0.0 works for us later on?

Reply:
To be clear, no problem merging with a dependency on openssl. I just also think it's really weird. Cartopy isn't doing anything SSL-specific here, it's just using `requests` to download some data.